server/public/index.html

```diff
@@ -6,12 +6,12 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Wicked Sales</title>
-  <link rel="shortcut icon" type="image/png" href="/images/favicon.png">
+  <link rel="shortcut icon" type="image/png" href="./images/favicon.png">
```

Comment:
You should be working on your front end at `http://localhost:3000` so the dot isn't necessary.